kanalony-storage-logic/src/main/scala/kanalony/storage/logic/queries/HourlyUserActivityPrtnEntryClstCountry.scala

```diff
@@ -25,8 +25,8 @@ class HourlyUserActivityPrtnEntryClstCountryQuery extends QueryBase[HourlyUserAc
     rawQueryResult
   }
 
-  override protected def getResultHeaders(): List[String] =  {
-    List(Dimensions.partner.toString, Dimensions.entry.toString, "metric", Dimensions.hour.toString, Dimensions.country.toString, "count")
+  override private[logic] def getResultHeaders(): List[String] =  {
```

Comment:
this method should get the query params as an argument so we can add the metric(s) to the headers list.